common/workunit/workunit.cpp

```diff
@@ -3639,6 +3640,10 @@ class CLockedWorkUnit : implements ILocalWorkUnit, implements IExtendedWUInterfa
             { return c->getScope(str); }
     virtual IStringVal & getSecurityToken(IStringVal & str) const
             { return c->getSecurityToken(str); }
+    virtual IStringVal & getSecuritySignature(IStringVal & str) const
```

Comment:
@RussWhitehead can you point out how these extracted signatures are being used?  Where the call is to validate them?  I'd rather see these direct set and get methods removed, but if they are really needed I can definitely be convinced.

Reply:
I am not sure at this time, since I still have a lot of code to add as subtasks. How about I open a Jira to re-evaluate removing this as I get closer to completion?

Reply:
How about flipping that and deleting for now, but adding later if needed?  Once we know where and why it's needed it will be an easy decision.

Reply:
@afishbeck Sounds reasonable